### PR TITLE
fix(nemesis) disable ValidateHintedHandoffShortDowntime until fixed

### PR DESCRIPTION
**Pull request by shoshan**
nemesis to be fixed by https://trello.com/c/Ru0T9Nmu/1239-fix-validatehintedhandoff-nemesis

Signed-off-by: Shoshan Dagany <shoshan.dagany@scylladb.com>

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/nemesis.py [hunk 1]
```diff
@@ -1751,8 +1751,8 @@ def repair_streaming_exists():
 
         self.log.debug("Execute a complete repair for target node")
         self.repair_nodetool_repair()
-
-    def disrupt_validate_hh_short_downtime(self):  # pylint: disable=invalid-name
+    #  Temporary disable due to https://trello.com/c/Ru0T9Nmu/1239-fix-validatehintedhandoff-nemesis
+    def _disrupt_validate_hh_short_downtime(self):  # pylint: disable=invalid-name
```
Comment (roydahan) [inline sdcm/nemesis.py:1755]:
```suggestion
    def disable_disrupt_validate_hh_short_downtime(self):  # pylint: disable=invalid-name
```

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -3482,7 +3482,7 @@ class ValidateHintedHandoffShortDowntime(Nemesis):
 
     @log_time_elapsed_and_status
     def disrupt(self):
-        self.disrupt_validate_hh_short_downtime()
+        self._disrupt_validate_hh_short_downtime()
```
Comment (roydahan) [inline sdcm/nemesis.py:3485]:
```suggestion
        self.disable_disrupt_validate_hh_short_downtime()
```

File: sdcm/nemesis.py [hunk 3]
```diff
@@ -1751,8 +1751,8 @@ def repair_streaming_exists():
 
         self.log.debug("Execute a complete repair for target node")
         self.repair_nodetool_repair()
-
-    def disrupt_validate_hh_short_downtime(self):  # pylint: disable=invalid-name
+    #  Temporary disable due to https://trello.com/c/Ru0T9Nmu/1239-fix-validatehintedhandoff-nemesis
```
Comment (roydahan) [inline sdcm/nemesis.py:1754]:
```suggestion
    # Temporary disable due to https://trello.com/c/Ru0T9Nmu/1239-fix-validatehintedhandoff-nemesis
    # TODO: Bentsi to fix this nemesis or investigate if it's a real scylla issue.
```

**Review by roydahan — CHANGES_REQUESTED** [review]
Please apply my suggestions into your commit.

File: sdcm/nemesis.py (hunk 1 above)
Comment (shoshan) [reply sdcm/nemesis.py:1755]: done

File: sdcm/nemesis.py (hunk 2 above)
Comment (shoshan) [reply sdcm/nemesis.py:3485]: done

File: sdcm/nemesis.py (hunk 3 above)
Comment (shoshan) [reply sdcm/nemesis.py:1754]: done

**Pull request merged**